src/monorobot.ml

```diff
@@ -56,7 +55,7 @@ let check_gh_action file json config secrets state =
     )
 
 let check_slack_action file secrets =
-  let data = Stdio.In_channel.read_all file in
+  let data = ExtLib.input_file file in
```

Comment:
Stdlib can do it too `In_channel.(with_open_bin file input_all)`

Reply:
`In_channel` is in OCaml from `4.14` onward, so we can probably use it if we bump up the require version?